internal/docs/readme.go

```diff
@@ -17,27 +17,50 @@ import (
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
-// ReadmeFile for the Elastic package
-const ReadmeFile = "README.md"
+// AreReadmesUpToDate function checks if all the .md readme file are up-to-date.
+func AreReadmesUpToDate() (string, string, error) {
+	packageRoot, err := packages.MustFindPackageRoot()
+	if err != nil {
+		return "", "", errors.Wrap(err, "package root not found")
+	}
 
-// IsReadmeUpToDate function checks if the README file is up-to-date.
-func IsReadmeUpToDate() (bool, error) {
-	logger.Debugf("Check if %s is up-to-date", ReadmeFile)
+	readmeFiles, err := ioutil.ReadDir(filepath.Join(packageRoot, "_dev", "build", "docs"))
+	if err != nil {
+		return "", "", nil
```

Comment:
error swallowed.

I think you need to be more specific and check if the directory doesn't exist (expected) or is it a different error (which we'd like to report).